wbooks/src/app/screens/Search/index.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { FlatList, ListRenderItem, View } from 'react-native';
+import { StackNavigationProp } from '@react-navigation/stack';
+import { LibraryStackParamList } from '@interfaces/navigatorParamList';
+import ItemBook from '@app/components/ItemBook';
+import SearchEmpty from '@app/components/SearchEmpty';
+import ViewText from '@app/components/ViewText';
+import { trimLineBreak } from '@utils/stringUtils';
+import Screens from '@constants/screens';
+import { Book } from '@interfaces/book';
+import SearchBooks from '@app/hooks/searchHook';
```

Comment:
```suggestion
import SearchBooks from '/hooks/searchHook';
```

If this doesnt exist add it to the config file